store/aciinfo.go

```diff
@@ -100,6 +105,29 @@ func GetACIInfosWithName(tx *sql.Tx, name string) ([]*ACIInfo, bool, error) {
 	return aciinfos, found, err
 }
 
+// getAciInfosWithDockerID returns all the ACIInfos with a given docker ID.
+// found will be false if no aciinfo exists.
```

Comment:
What's the reason for additional boolean return value instead of simply returning an empty list?